hashset.py

```diff
@@ -0,0 +1,67 @@
+# // Time Complexity : O(n/k) n is no.of all possible values and k is no.of buckets.
+# // Space Complexity : O(m+k) m is predefined buckets and k is inserted values.
```

Comment:
Please try to refactor your code to get  O(1) accessing time as the main feature if hastset is O(1) time complexity.